source/direct_debit/index.html.md.erb

```diff
@@ -0,0 +1,324 @@
+---
+title: Direct Debit
+weight: 75
+---
+
+# Direct Debit
+
+Before reading this section, you should be familiar with [making
+payments](/making_payments). You should also
+have [integrated your service with the GOV.UK Pay
+API](/integrate_with_govuk_pay/#how-to-integrate-with-the-gov-uk-pay-api).
+
+### Synchronous and asynchronous payment methods
+
+Card payments are a synchronous payment method. When a user
+completes a card payment using GOV.UK Pay, the platform immediately confirms
+to your service whether the payment has succeeded or failed. If a user
+pays by card, you can immediately release your service to the user .
+
+Direct Debit payments are an asynchronous payment method. When a user
+completes a Direct Debit payment process using GOV.UK Pay, the platform makes
+a request to their bank to set up the Direct Debit mandate and transfer the
+funds. It takes 3-4 working days to confirm the success of the payment.
+
+## Get started
+
+To use the GOV.UK Pay Direct Debit feature, you must request a GOV.UK Pay
+Direct Debit test account [from our support
+team](/support_contact_and_more_information/#contact-us).
+
+If you want to add Direct Debit to an existing integration, you should
+[contact us](/support_contact_and_more_information/#contact-us) to discuss
+this.
+
+### Set up an account with the Direct Debit supplier
+
+Select your account in the __My services__ page of your GOV.UK Pay Direct
+Debit account then follow the on-screen instructions.
```

Comment:
it will just be their general account rather than payment type specific, so this should say GOV.UK Pay account